Jenkinsfile

```diff
@@ -6,158 +6,76 @@ import groovy.json.*
 // 'global' var to store git info
 def scmVars
 // 'global' var to store application version
-def appversion
+def commitid
 
 try { // massive try{} catch{} around the entire build for failure notifications
-
-node('master'){
-    scmVars = checkout scm
-    scmVars.GIT_BRANCH_NAME = scmVars.GIT_BRANCH.split('/')[-1]  // origin/pr/1234 -> 1234
-
-    // setting build display name
-    def branch = scmVars.GIT_BRANCH_NAME
-    if ( branch == 'master' ) {
-        echo 'Building master'
-        currentBuild.displayName = 'master'
-    }
-    else if (branch ==~ /[0-9]+/) {
-        def pagureUrl = "https://pagure.io/freshmaker/pull-request/${branch}"
-        def pagureLink = """<a href="${pagureUrl}">PR-${branch}</a>"""
-        try {
-            def response = httpRequest "https://pagure.io/api/0/freshmaker/pull-request/${branch}"
-            // Note for future use: JsonSlurper() is not serialiazble (returns a LazyMap) and
-            // therefore we cannot save this back into the global scmVars. We could use
-            // JsonSlurperClassic() which returns a hash map, but would need to allow this in
-            // the jenkins script approval.
-            def content = new JsonSlurper().parseText(response.content)
-            pagureLink = """<a href="${pagureUrl}">${content.title}</a>"""
-        } catch (Exception e) {
-            echo 'Error using pagure API:'
-            echo e.message
-            // ignoring this...
-        }
-        echo "Building PR #${branch}: ${pagureUrl}"
-        currentBuild.displayName = "PR #${branch}"
-        currentBuild.description = pagureLink
-    }
-}
-
     timestamps {
+        node('docker') {
+            stage('Prepare') {
+                scmVars = checkout scm
 
-node('fedora-29') {
-    stage('Prepare') {
-        checkout scm
-        // Install requirements for testing harness
-        sh '''
-        sudo dnf -y install python3-tox python3-flake8 \
-        rpm-libs gcc rpm-devel krb5-devel openldap-devel koji \
-        `cat yum-packages.txt`
-        '''
-        // The fedora-29 node has an older version of setuptools that causes
-        // tox to fail processing the dependencies
-        sh 'sudo dnf update -y python3-setuptools'
-    }
-    stage('Determine Version') {
-        // TODO: Set appversion as a label on the image?
-        appversion = sh(returnStdout: true, script: """
-            printf `python setup.py -V`-`git log -1 --pretty=format:%ct`
-        """).trim()
-    }
-    stage('Run unit tests') {
-        sh 'tox'
```

Comment:
Why removing this part and the documentation part? I thought it was useful.

Reply:
Because Travis CI now runs these tests and build+publish the docs: https://github.com/redhat-exd-rebuilds/freshmaker/commit/5dd27852c5cd4738fd0a29ed1d6509bfec816d9b